pom.xml

```diff
@@ -124,16 +124,9 @@
       <dependency>
         <groupId>org.spockframework</groupId>
         <artifactId>spock-core</artifactId>
-        <version>2.2-M3-groovy-4.0</version>
+        <version>2.4-M1-groovy-4.0</version>
         <scope>test</scope>
       </dependency>
-      <!--If not specified, the 2.7.6 spring-boot-starter-parent dependency will introduce transitively the 1.30 version of snakeyaml, for which a multitude of security concerns are reported
```

Comment:
Does spring-boot 3 fix this problem?

Reply:
Thanks to your review i doublechecked the dependency tree and I was wrong to assume that this was addressed. [Apparently](https://stackoverflow.com/questions/75870282/is-snakeyaml-2-0-added-in-the-new-spring-boot-versions) spring boot stayed with 1.33 to avoid dependency issues. 
Therefore i readded this dependency 👍